### PR TITLE
NULLS FIRST and NULLS LAST operators

### PR DESCRIPTION
**Pull request by amybalmforth**
NULLS FIRST and NULLS LAST operators added.

### CODE REVIEW

File: tests/test_logic.py [hunk 1]
```diff
@@ -103,6 +103,8 @@ def test_render(template, var, expected_query, expected_params):
         (lambda: V('x').comma(V('y')), 'x, y'),
         (lambda: V('a').asc(), 'a ASC'),
         (lambda: V('a').desc(), 'a DESC'),
+        (lambda: V('a').nulls_first(), 'a NULLS FIRST'),
```
Comment (samuelcolvin) [inline tests/test_logic.py:106]: can we just try `V('a').desc().nulls_first()` and check the precedence order makes sense.

Comment (amybalmforth) [reply tests/test_logic.py:106]: Sure - have updated

**Comment by samuelcolvin** [comment]
awesome, thank you.

Do you want me to do a new release with this, or are there any other changes you would like to make?

**Comment by amybalmforth** [comment]
Thanks a lot! No, no more changes - a release would be great!

**Pull request merged**